source/transactions/tests/bulk.yml

```diff
@@ -70,19 +70,21 @@ tests:
             - name: deleteOne
               arguments:
                 filter: {_id: 4}
-            - name: deleteMany
-              arguments:
-                filter: {_id: {$gte: 6}}
             - name: updateMany
               arguments:
                 filter: {_id: {$gte: 2}}
                 update: {$set: {z: 1}}
+            # Keep deleteMany segregated from deleteOne, so that drivers that aren't able to coalesce
+            # adjacent updates and replaces into a single delete command will still pass this test
```

Comment:
"updates and replaces" -> "deletes and deleteManys"

Reply:
Shoot, I could have sworn I updated all of that